internal/pkg/cli/svc_deploy.go

```diff
@@ -555,6 +566,20 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 	}
 
 	if err := o.svcCFN.DeployService(os.Stderr, conf, awscloudformation.WithRoleARN(o.targetEnvironment.ExecutionRoleARN)); err != nil {
+		if _, ok := err.(*awscloudformation.ErrChangeSetEmpty); ok {
```

Comment:
nit: to be safe:
```suggestion
		var errEmptyCS *awscloudformation.ErrChangeSetEmpty
		if errors.As(err, &errEmptyCS) {
```

Reply:
Hmm why is it safer?

Reply:
in case the error gets wrapped in the `cloudformation` package like this:
```go
fmt.Errorf("my wrapped error: %w", &ErrChangeSetEmpty{})
```
Then `errors.As` will catch that the err is composed of an empty change set up the chain. On the other hand this check will only compare with the immediate error `"my wrapped error"` which isn't a `ErrChangeSetEmpty`